src/databricks/labs/lsql/dashboards.py

```diff
@@ -51,16 +65,53 @@ def as_dict(self) -> dict[str, str]:
 
 @dataclass
 class WidgetMetadata:
-    order: int
-    width: int
-    height: int
+    path: Path
+    order: int = 0
+    width: int = 0
+    height: int = 0
+    id: str = ""
+
+    def __post_init__(self):
```

Comment:
Don't do dataclass here, do a normal class

Reply:
Done: https://github.com/databrickslabs/lsql/pull/154/commits/01c814ad55a27a302aa2f647cf9d28105e66b92d. 

Intuively I prefer a data class here. It makes sense to me for `Metadata` to be represented by a `dataclass`. We use a `dataclass` more often for similar objects, like the `DashboardMetadata` and `Specs`. Though, I do recognize that the `__post_init` and other methods make this class (maybe) too complicated to remain a `dataclass`. 

Note that a couple times `dataclasses.<method>` were used, which I had to replace when refactoring this to a normal class. 

In any case, we can always revert.